app/code/Magento/CatalogSearch/Model/Adapter/Mysql/Aggregation/DataProvider.php

```diff
@@ -6,65 +6,58 @@
 namespace Magento\CatalogSearch\Model\Adapter\Mysql\Aggregation;
 
 use Magento\Catalog\Model\Product;
-use Magento\CatalogInventory\Model\Stock;
-use Magento\Customer\Model\Session;
+use Magento\CatalogSearch\Model\Adapter\Mysql\Aggregation\DataProvider\SelectBuilderForAttribute;
```

Comment:
Need to check why do we do not use `show of stock products` option in original `Magento\CatalogSearch\Model\Adapter\Mysql\Aggregation` class 
Looks like we have bug in core code

Reply:
Yes. I tested it and it seems like a bug. What should be my next steps?